src/components/Modal.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
Is this file being used anywhere?

Reply:
I believe I removed it locally but didnt push those changes yet. The code for the modal got moved to "GridItem.js"